cmd/notary/config.json

```diff
@@ -0,0 +1,7 @@
+{
+	"remote_server": {
+		"url": "https://localhost:4443",
```

Comment:
Setting this to "https://notary-server:4443" might make this more friendly for those whose docker hosts are not on localhost (since the README already suggests adding a `notary-server` entry to /etc/hosts).